api/v1beta1/activemqartemis_types.go

```diff
@@ -377,6 +377,16 @@ type DeploymentPlanType struct {
 	// Specifies the container security context
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Container Security Context"
 	ContainerSecurityContext *corev1.SecurityContext `json:"containerSecurityContext,omitempty"`
+	// Specifies additional Volumes to be attached to the broker pods
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Volumes"
+	ExtraVolumes []corev1.Volume `json:"extraVolumes,omitempty"`
+	// Specifies mount options for extraVolumes
+	// +optional
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Mount Options"
+	ExtraVolumeMounts []*corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
```

Comment:
The field `ExtraVolumeMounts` could be an array of [VolumeMount](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#volumemount-v1-core) as in the [container](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#container-v1-core).
```suggestion
	ExtraVolumeMounts []corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
```

Reply:
+1